packages/shared/src/modules/auth/libs/enums/auth-api-path.enum.ts

```diff
@@ -1,4 +1,5 @@
 const AuthApiPath = {
+	AUTHENTICATED_USER: "/authenticated-user",
 	ROOT: "/",
```

Comment:
We have connected tasks and Yelyzaveta asked me to remove ROOT. https://github.com/BinaryStudioAcademy/bsa-2024-gitfit/pull/49#discussion_r1725518554
Can you do this as well, so that later it does not accidentally return?

Reply:
I’m keeping an eye on your PR too, so ROOT definitely won’t return)